src/main/java/com/rackspacecloud/metrics/queryservice/services/QueryServiceImpl.java

```diff
@@ -67,7 +69,7 @@ public QueryServiceImpl(
         catch(Exception e) {
             String errMsg = String.format(
                     "Failed to get routes for tenantId [%s] and measurement [%s]", tenantId, measurement);
-            LOGGER.error(errMsg, e);
+            log.error(errMsg, e);
```

Comment:
Instead of using `String.format` could do
```suggestion
            log.error("Failed to get routes for tenantId [{}] and measurement [{}]", tenantId, measurement, e);
```

Reply:
Don't know where my comment went. :-) so writing this again. 
I am using same string twice, hence setting it only one place using `String.format()`. Also, this will be called only when exception is thrown.